plansys2_executor/src/plansys2_executor/ExecutorClient.cpp

```diff
@@ -192,12 +192,9 @@ std::vector<parser::pddl::tree::Goal> ExecutorClient::getOrderedSubGoals()
   }
 
   if (future_result.get()->success) {
-    for (std::string goal_str : future_result.get()->sub_goals) {
-      parser::pddl::tree::Goal new_goal(goal_str);
-      ret.push_back(new_goal);
-    }
+    ret = future_result.get()->sub_goals;
   } else {
-    RCLCPP_INFO_STREAM(
```

Comment:
I think this change to the log statement severity should be undone.
original PR making the change to INFO: https://github.com/IntelligentRoboticsLabs/ros2_planning_system/pull/109